Source/Core/Core/PowerPC/JitCommon/Jit_Util.h

```diff
@@ -46,6 +46,43 @@ static const int FARCODE_SIZE_MMU = 1024 * 1024 * 48;
 static const int TRAMPOLINE_CODE_SIZE = 1024 * 1024 * 8;
 static const int TRAMPOLINE_CODE_SIZE_MMU = 1024 * 1024 * 32;
 
+// Stores information we need to batch-patch a MOV with a call to the slow read/write path after
+// it faults
+class BackPatchInfo {
```

Comment:
This doesn't have any invariants, it should be a struct. Also it should be final.